server/api.go

```diff
@@ -400,6 +400,8 @@ func (p *Plugin) completeConnectUserToGitlab(c *Context, w http.ResponseWriter,
 		}
 	}
 
+	p.TrackUserEvent("account_connected", userID, nil)
```

Comment:
@mickmister is this the most proper/accurate place or should it be within the `else` block with the welcome message?